Library/Homebrew/rubocops/mixins/on_formula_class.rb

```diff
@@ -0,0 +1,18 @@
+module RuboCop
+  module Cop
+    # Helper module for checking formula class
+    module FormulaDef
+      def file_inside_formula_root?
+        return false unless processed_source.path =~ %r{#{ENV["HOMEBREW_LIBRARY"]}\/Taps\/homebrew}
```

Comment:
Yeh, that sounds like a sensible default. Don't need `\/` inside a `%r`, though!